themes/SuiteP/include/ListView/ListViewPaginationBottom.tpl

```diff
@@ -58,7 +58,7 @@
 
 						{sugar_action_menu id=$link_action_id params=$actionsLink}
 
-                        { if $actionDisabledLink ne "" }<div class='selectActionsDisabled' id='select_actions_disabled_{$action_menu_location}'>{$actionDisabledLink}<span class='ab'></span></div>{/if}
+                        { if $actionDisabledLink ne "" }<div class='selectActionsDisabled' id='select_actions_disabled_{$action_menu_location}'>{$actionDisabledLink}</span></div>{/if}
```

Comment:
wrong html struct: missing `<span>` open tag. please make sure you use correct html struct everywhere

Reply:
Removed